src/server/mod.rs

```diff
@@ -92,10 +92,23 @@ impl AutopushServer {
         }
     }
 
-    pub fn start(&self) {
+    pub fn start(&self) -> Option<sentry::internals::ClientInitGuard> {
         logging::init_logging(!self.opts.human_logs).expect("init_logging failed");
         let handles = Server::start(&self.opts).expect("failed to start server");
         self.shutdown_handles.set(Some(handles));
+        if let Ok(dsn) = env::var("SENTRY_DSN") {
```

Comment:
nit: we could just store the guard inside of AutopushServer. I suppose this could all go in new above to make that easier